src/components/link-detail/index.js

```diff
@@ -197,6 +199,11 @@ export class LinkDetail extends React.Component {
           >{link.lastWebhookSync && link.lastWebhookSync.status === 'TRIGGERED' ? 'Waiting...' : 'Resync'}</Button>
         </div>
 
+        { link.lastSyncedAt ?
```

Comment:
Can you format `{ link.lastSyncedAt ? <div className="link-detail-row">` on one line?

Also, why is the ternary required? I think `link.lastSyncedAt` should always exists, if I remember right.

Reply:
I think the same form is used for creating the link. But if its always available it should not be required.
Though I haven't gone through the server and worker code yet!

Reply:
Oh, that's true. When creating the link I'm not sure that the last synced time will exist. A ternary probably should be here.